packages/ow-db/prisma/schema.prisma

```diff
@@ -29,3 +29,18 @@ model Company {
   website     String?
   location    String?
 }
+
+model Event {
```

Comment:
Probably be smart with a 

1. event status, such as Open event, Attendance event, Infinite slots, TBA (we have a lot of TBA events)
2. Eventually, relations to attendance, payment and the users (attendee)